ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Configuration.scala

```diff
@@ -15,12 +15,14 @@ final case class Configuration(
     generation: Long,
     /** The time model of the ledger. Specifying the time-to-live bounds for Ledger API commands. */
     timeModel: TimeModel,
+    /** The maximum time window during which commands can be deduplicated. */
+    maxCommandTtl: Duration,
 )
 
 object Configuration {
   import com.daml.ledger.participant.state.protobuf
 
-  val protobufVersion: Long = 1L
+  val protobufVersion: Long = 2L
```

Comment:
Consider using aliases  for version when a specific feature was introduced and use that alias for checking in the decoding.

```scala
val maxCommandTtlVersion = 2L
```